Move gunbag type from inheritance to config property (#7022)
acemod/ACE3: docs/wiki/framework/gunbag-framework.md

---
layout: wiki
title: Gunbag Framework

> description: Explains how to set-up gunbags

Reviewer: ```suggestion
description: Explains how to set up gunbags
```

Author: I copied that from grenades. Might also wanna change there?

group: framework
parent: wiki
order: 7
mod: ace
version:
  major: 3
  minor: 13
  patch: 0
---

## 1. Overview

> ACE provides a gunbag thing that you can put any gun inside.

Reviewer: ```suggestion
ACE Gunbag provides a framework that allows users to enable putting a gun inside a backpack.
```

## 2. Config Values

> The config entry `ace_gunbag` needs to be set to 1 to enable a backback to be a gunbag

Reviewer: ```suggestion
The `ace_gunbag` config entry needs to be set to `1` to enable a backpack to be a gunbag.
```

---
layout: wiki
title: Gunbag Framework
description: Explains how to set-up gunbags
group: framework
parent: wiki
order: 7
mod: ace
version:
  major: 3
  minor: 13
  patch: 0
---

## 1. Overview

ACE provides a gunbag thing that you can put any gun inside.

## 2. Config Values

The config entry `ace_gunbag` needs to be set to 1 to enable a backback to be a gunbag

```cpp
class Bag_Base;
class ace_gunbag: Bag_Base {

>     _generalMacro = QUOTE(ADDON);

Author: Mh? I don't see it. It's 4 space for me.

>     _generalMacro = QUOTE(ADDON);

Author: I've removed _generalMacro from the doc as we don't need it anyway.

>     _generalMacro = QUOTE(ADDON);

Reviewer 2: Idk, not all lines were indented the same when I looked at diff.
